styles.css

```diff
@@ -18,6 +23,21 @@
   --coral: #f28373;
 }
 
+body {
+  background-color: var(--primary-background);
+  color: var(--primary-color);
+  font-family: "Poppins", sans-serif;
+}
+
+main > h1 {
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 48px;
+  margin-top: 67px;
```

Comment:
Decrease margin-top value. I think somehting closer to 35px should be good. Tinker a little bit, choose precise value based on own observations.